WinUIGallery/WinUIGallery.csproj

```diff
@@ -312,5 +312,6 @@
                 <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
             </_NoneWithTargetPath>
         </ItemGroup>
-    </Target> 
+    </Target>
+    <ProjectExtensions><VisualStudio><UserProperties datamodel_4iconsdata_1json__JsonSchema="IconsDataSchema.json" /></VisualStudio></ProjectExtensions> 
```

Comment:
nit: can this be separated out into different lines?

Reply:
It might be accidentally added by Visual Studio to cache the scheme. I didn't even know it exist. It's now removed.